### ISSUE TITLE
FreeBSD service module rc.conf entries

### ISSUE THREAD

**Issue by elpek** [ansible/ansible#4867]
Hello,

The problem started with release 1.3.3 and is still present in version 1.3.4.

Using service module in playbook:

service: name=example_service enabled=yes

Produced entry in rc.conf:

example_service="YES"

Correct form is:

example_service_enable="YES"

**Comment by jctanner** [comment]
@elpek the string for the entry comes from the service command and the service it's self...

```
[root@freebsd ~]# /usr/sbin/service apache22 rcvar
# apache22
#
apache22_enable="YES"
#   (default: "")

[root@freebsd ~]# fgrep apache22 /etc/rc.conf 
apache22_enable="YES"
```

I'm not sure if example_service is your real service, but can you show me the rcvar that comes from it?

```
 /usr/sbin/service <servicename> rcvar
```

**Comment by elpek** [comment]
@jctanner, ok, you made it all clear for me, thank you very much. The reason I assumed the server module not to work properly was that I have tested  it with a playbook which was supposed to set 'nfsv4_server_enable="YES"`. Since the nfsv4_server doesn't have any valid rc script and is in fact just a kind of a flag the service module returned an empty string. Now from what I see service module works a bit different, it relays on rcvar value rather than just append "_enable" suffix to whatever is defined in playbook. 

My fault I did not test some other services prior to submiting the issue - of course it can be closed.
